datasets/gov/lsd/statistika/bvp_islaidu_metodu.csv

```diff
@@ -11,7 +11,20 @@ id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri
 ,,,,,islyginimas_id,string,,,,4,open,,Išlyginimas: identifikacinis numeris,
 ,,,,,laikotarpis,date,D,,,4,open,,Laikotarpis,
 ,,,,,laikotarpis_id,string,,,,4,open,,Laikotarpis: identifikacinis numeris,
-,,,,,matavimo_vienetai,number,,,,4,open,,Matavimo vienetai,
+,,,,,matavimo_vienetai,string,,,,4,open,,Matavimo vienetai,
+,,,,,matavimo_vienetai_id,string,,,,4,open,,Matavimo vienetai: identifikacinis numeris,
+,,,,,verte,number,,,,4,open,,Skaitinė vertė,
+,,,,,atnaujinta,date,D,,,4,open,,Paskutinis atnaujinimas,
+,,,,,,,,,,,,,,
+,,,,S7R192_M2110107_1,,,vda_id,,,4,,,"“BVP išlaidų metodu, to meto kainomis, ketvirtinis""","“Statistinės informacijos aprašymas: bendrasis vidaus produktas (BVP) – vienas iš pagrindinių nacionalinių sąskaitų sistemos (NSS) rodiklių, apibūdinančių šalies ekonomikos išsivystymo lygį. BVP, jo kitimo tempas ir bendrosios nacionalinės pajamos (BNP) plačiai naudojami tarptautiniams palyginimams ir ekonominei analizei atlikti.  Klasifikatorių nėra. Statistinio stebėjimo vienetas: nacionalinės sąskaitos, apimančios ekonomiką kaip visumą, kurią sudaro skirtingi instituciniai vienetai. Statistinę (tiriamąją) visumą sudaro visi statistiniai stebėjimo vienetai rezidentai.  Geografinė aprėptis - šalies ekonominė teritorija.  Laiko aprėptis nuo 1995 m I ketvirčio. Matavimo vienetas (-ai): BVP to meto kainomis – mln. EUR ir indeksai – proc.; Ataskaitinis laikotarpis: kalendorinis ketvirtis """
```

Comment:
čia irgi gerai padaryta. Tik, kaip ankstesnėje ADSA minėjau, reiktų nuimti kabutes, kur nereikia, 19 ir 16 eilutėse, kad Saugykloje pavadinimas matytųsi be kabučių.